src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -4837,6 +4837,9 @@ unsafe public static int Main()
                 // (7,34): error CS1002: ; expected
                 //         int *pp = stackalloc int 30; 
                 Diagnostic(ErrorCode.ERR_SemicolonExpected, "30").WithLocation(7, 34),
+                // (6,18): error CS1525: Invalid expression term 'stackalloc'
+                //         int *p = stackalloc int (30); 
```

Comment:
Duplicate error as well.

Reply:
the error is correct here - stackalloc is a target of invocation here, that is not valid location.

---
In reply to: [139548763](https://github.com/dotnet/roslyn/pull/22162#discussion_r139548763) [](ancestors = 139548763)